Lambda Hot Reloading not updating function when not using Docker desktop (#1502)
localstack/docs: content/en/user-guide/lambda-tools/hot-reloading/index.md

**Layer limit with hot reloading for layers:**
When hot reloading is active for a Lambda layer (Pro), the function can use at most one layer.

{{< callout >}}

> The file sharing mechanism in Rancher Desktop or Colima distributions requires additional configuration for Lambda to be reloaded automatically.

Reviewer: ```suggestion
Configuring the file sharing mechanism in Rancher Desktop or Colima distributions is necessary to enable hot reloading for Lambda.
```

* For Rancher Desktop it is required to set the configuration `LAMBDA_DOCKER_FLAGS=-e LOCALSTACK_FILE_WATCHER_STRATEGY=polling`.

> * In case of Colima, we recommend starting it with Virtiofs mount type like this: `colima start --vm-type vz --mount-type virtiofs`.

Reviewer: ```suggestion
* For Colima, it is required to start with the Virtiofs mount type: `colima start --vm-type vz --mount-type virtiofs`.
```
Is it required or recommended? Is it necessary to use the `virtiofs` to enable it?
